package.json

```diff
@@ -31,6 +31,7 @@
     "@material-ui/core": "^3.9.1",
     "@material-ui/icons": "^3.0.2",
     "@researchgate/react-intersection-observer": "^0.7.4",
+    "acorn": "6.1.1",
```

Comment:
Where is acorn being used?

Reply:
It's a peer dependency that was uncovered when I was forced to run things on a new machine.

Reply:
I'm not sure i understand that. Wouldn't the package list that as a dependency if it was needed? Why wouldn't our CI/CD catch that?